### ISSUE TITLE
readme on compile the project in android studio

### ISSUE THREAD

**Issue by lzzhang75** [ayuso2013/OpenCVSamples#1]
could you share instruction on how to compile this project in android studios? I tried to compile, but received a bunch of errors.

**Comment by ayuso2013** [comment]
Hi

Did you import the code into Android Studio?

2017-03-27 7:40 GMT+02:00 lzzhang75 <notifications@github.com>:

**Comment by lzzhang75** [comment]
Yes, I did. However, there were one error after another. Would simply import / open the project in Android Studio work? I am using Android Studio 2.3. Thanks

**Comment by lzzhangar** [comment]
I created a project from scratch and import the code in android studio, it works now.
